_docs/structure-of-a-contract.md

```diff
@@ -5,8 +5,12 @@ permalink: /docs/structure-of-a-contract/
 ---
 
 In Solidity, contracts are what classes are in object-oriented languages.
-They can contain so-called state variables, which are permanently stored
-in storage together with the contract and functions, the entry pointers
-into operating on these state variables. Apart from state variables, there are
-also local variables declared inside functions, whose content is cleared
-as soon as control flow returns from the function.
+They can contain **state variables**, **functions**, and **events**.
```

Comment:
There are also struct declarations (they might be allowed outside at some point) enum declarations (dito) and modifier declarations.